builder-model/src/main/java/com/google/appengine/gradle/model/AppEngineModel.java

```diff
@@ -34,4 +34,5 @@
     public File getWebAppDir();
     public String getAppEngineSdkRoot();
     public AppCfgOptions getAppCfgOptions();
+    public String getGeneratedDir();  // FIXME - Note above says update the builder model version - not sure where that is.
```

Comment:
Lets avoid modifying the model. I don't think we're going to publish a new model at the moment.